tracer/build/_build/Build.Steps.cs

```diff
@@ -92,6 +92,7 @@ partial class Build
 
     bool IsArm64 => RuntimeInformation.ProcessArchitecture == Architecture.Arm64;
     string LinuxArchitectureIdentifier => IsArm64 ? "arm64" : TargetPlatform.ToString();
+    string OsxArchitectureIdentifier => IsArm64 ? "arm64" : "x64";
```

Comment:
Could we merge `OsxArchitectureIdentifier` and `LinuxArchitectureIdentifier`? Looks like they will always have the same value.

Reply:
done in [eaa7e09](https://github.com/DataDog/dd-trace-dotnet/pull/3279/commits/eaa7e09141471930c52e8fb97fcd487646d82f17)